src/CommunityToolkit.Aspire.Hosting.Ollama/PublicAPI.Shipped.txt

```diff
@@ -11,10 +11,8 @@ Aspire.Hosting.ApplicationModel.OpenWebUIResource.ConnectionStringExpression.get
 Aspire.Hosting.ApplicationModel.OpenWebUIResource.OpenWebUIResource(string! name) -> void
 Aspire.Hosting.ApplicationModel.OpenWebUIResource.PrimaryEndpoint.get -> Aspire.Hosting.ApplicationModel.EndpointReference!
 Aspire.Hosting.OllamaResourceBuilderExtensions
-static Aspire.Hosting.OllamaResourceBuilderExtensions.AddModel(this Aspire.Hosting.ApplicationModel.IResourceBuilder<Aspire.Hosting.ApplicationModel.OllamaResource!>! builder, string! modelName) -> Aspire.Hosting.ApplicationModel.IResourceBuilder<Aspire.Hosting.ApplicationModel.OllamaResource!>!
```

Comment:
We need to leave this method here and instead it'll get marked with REMOVED in the unshipped file.

@Alirexaa we should document this in the contributing guide better.